common/azure-common/src/main/java/com/azure/common/implementation/SwaggerMethodParser.java

```diff
@@ -526,4 +519,31 @@ private String applySubstitutions(String originalValue, Iterable<Substitution> s
 
         return result;
     }
+
+    private HashMap<Integer, UnexpectedException> processUnexpectedResponseExceptionTypes() {
+        HashMap<Integer, UnexpectedException> exceptionHashMap = new HashMap<>();
+
+        for (UnexpectedResponseExceptionType exceptionAnnotation : unexpectedResponseExceptionTypes) {
+            UnexpectedException exception = new UnexpectedException(exceptionAnnotation.value());
+            if (exceptionAnnotation.code().length == 0) {
+                if (defaultException == null) {
+                    defaultException = exception;
+                } else {
+                    throw new InvalidUnexpectedResponseAnnotationsException(fullyQualifiedMethodName);
+                }
+            } else {
+                for (int statusCode : exceptionAnnotation.code()) {
+                    if (exceptionHashMap.putIfAbsent(statusCode, exception) != null) {
+                        throw new InvalidUnexpectedResponseAnnotationsException(fullyQualifiedMethodName, statusCode);
+                    }
+                }
+            }
+        }
```

Comment:
I think we should change this. Don't throw exceptions in either case - just take the most recent value as the definitive one and replace the previous one.

Reply:
Removed exception throws